compiler/make/make.cpp

```diff
@@ -177,6 +182,9 @@ static long long get_imported_header_mtime(const std::string &header_path, const
   return 0;
 }
 
+// todo:k2 review pch-related things for
+// k2-component mode
+
```

Comment:
Let's remove such TODOs. In the single stage build it's fully supported

Reply:
remove